### ISSUE TITLE
After filling in a form - set goals up / reminder for next time

### ISSUE THREAD

**Issue by haydenrou** [hay-i/chronologger#31]

**Comment by jonnyjones99** [comment]
Do we want to integrate some sort of SMTP email reminders?

Might be interesting to write our own from scratch using (this)[https://github.com/emersion/go-smtp]

Or just some out of the box service. I've used (Mailgun)[https://www.mailgun.com] loads personally and it's great and got a free plan.

**Comment by jonnyjones99** [comment]
Also thinking more about this we will need some sort of emails for authentication, confirm account reset password etc

**Comment by haydenrou** [comment]
Yeah email notifications are definitely on the cards but I'd push back on it for the moment. At the least, we want to set up user auth first - it can come a bit later down the line
